README.md

```diff
@@ -260,12 +258,23 @@ sc <- spark_connect(master = "local", version = "2.0.0")
 	```
 	
 4. Note, the `spark_home` parameter in `spark_connect` defaults to the `SPARK_HOME` environment variable. If `SPARK_HOME` is defined it will be always be used unless the `version` parameter is specified to force the use of a locally installed version.
-5. Run the following:
+5. Run the following to create a Spark connection using the default IP and port:
 
 	``` r
 	sc <- spark_connect(master = "local")
 	```
 
+### Spark Connection on Custom Port
+
+RSparkling does not expose setters and getters for specifying configuration options. You must specify the Spark configuration options directly, for example:
+
+``` r
+config=spark_config()
+config=c(config,list("spark.ext.h2o.node.port.base"="55555", "spark.ext.h2o.client.port.base"="44444"))
```

Comment:
I would document here as well that `spark.ext.h2o.node.port.base` affects the worker nodes and `spark.ext.h2o.client.port.base` the client.